Google.Api.Generator.Rest.Tests/GoldenTestData/Google.Apis.Assuredworkloads.v1/Google.Apis.Assuredworkloads.v1.cs

```diff
@@ -644,13 +644,71 @@ public ListRequest(Google.Apis.Services.IClientService service, string parent) :
                         [Google.Apis.Util.RequestParameterAttribute("filter", Google.Apis.Util.RequestParameterType.Query)]
                         public virtual string Filter { get; set; }
 
-                        /// <summary>The end of the time window.</summary>
+                        private object _intervalEndTime;
+
+                        /// <summary>
+                        /// String representation of <see cref="IntervalEndTimeDateTimeOffset"/>, formatted for
+                        /// inclusion in the HTTP request.
+                        /// </summary>
                         [Google.Apis.Util.RequestParameterAttribute("interval.endTime", Google.Apis.Util.RequestParameterType.Query)]
-                        public virtual object IntervalEndTime { get; set; }
+                        public virtual string IntervalEndTimeRaw { get; private set; }
```

Comment:
Thinking more, do we even need this to be public? I like very much to have this property to apply the attribute to, and we control serialization better. But would users need it at all?

Reply:
Users don't, but we fetch this via reflection, and in some cases I can imagine things breaking if we access internal or private properties via reflection in a different assembly. If we're already doing that elsewhere, I'm happy to include it here too - but I suspect we don't.

Reply:
Righto. Had you looked enough to be really happy for me to merge? This all feels so fiddly I don't want to merge if you'd like any more time at all. (I'm aware you've LGTM'd, I'm just being super-cautious.)

Reply:
Yep, I think this is fine. I've tried to run over all possible scenarios and I think people that may run into trouble are those that attempt to mix old and new properties. That should be rare, and there's really no reason that I can think so to do so anyway.

Reply:
Great, thanks. Merging.